tests/test_cli.py

```diff
@@ -835,6 +839,16 @@ def test_bad_dfe(self):
                 f"DFE 'HomSap/XXX' not in catalog ({avail_dfes_str})"
             )
 
+    def test_bad_annotation(self):
+        species = stdpopsim.get_species("HomSap")
+        with mock.patch("stdpopsim.cli.exit", autospec=True) as mocked_exit:
+            cli.get_annotation_wrapper(species, "666_foo_666")
+            available_annots = [dm.id for dm in species.annotations]
+            avail_annots_str = ", ".join(available_annots)
+            mocked_exit.assert_called_once_with(
+                f"Annotations 'HomSap/666_foo_666' not in catalog ({avail_annots_str})"
+            )
```

Comment:
add testes that would lead to the error messages you have included here.
But as you suggested to me last time, we can add most of the tests into test_slim_engine.py

Reply:
👍

Reply:
Hm - why are those CLI tests in test_slim.py, not test_cli.py?

Reply:
good question, but `test_slim.py` has it's own `TestCLI` class. let's refactor that in a separate PR